stable/appmesh-controller/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: appmesh-controller
 description: App Mesh controller Helm chart for Kubernetes
-version: 1.4.4
+version: 1.4.5
```

Comment:
Looks like current version is 1.4.6 as of April 13: https://github.com/aws/eks-charts/blob/43abf55028ff65874fcd951ea7dc2f8170278c4d/stable/appmesh-controller/Chart.yaml#L4

You need to rebase and up this to 1.4.7 :)

Reply:
@jaypipes I'm curious, since it's just the chart version, can't the maintainers do the rebase and patch the chart version?

Reply:
@gitfool yes, the maintainers could do the rebase and patch the chart version.

Reply:
Normally I would do this if GH would let me rebase in the UI... not in front of my desktop at the moment so can't use command line...

Reply:
FWIW, it does allow rebase now; see [more ways to keep your pull request branch up-to-date](https://github.blog/changelog/2022-02-03-more-ways-to-keep-your-pull-request-branch-up-to-date).

Reply:
Let me see if I can take care of rebasing. Thanks @jaypipes for giving me write access

Reply:
I did the [rebase](https://github.com/bendu/eks-charts/commit/1b55ad0693d9cca5b176af16c6100a90c75a706e) Should I just push the change straight to master or make another PR?

Reply:
@bendu do you have enough permission [allowing changes to a pull request branch created from a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)? In which case you would update the original branch and force push before approving and merging.